framework/src/modules/network/defaults/config.js

```diff
@@ -14,6 +14,8 @@
 
 'use strict';
 
+const LiskP2P = require('@liskhq/lisk-p2p');
```

Comment:
I think it shouldn't read the library from this, this is more of a application constants
If it doesn't require default value (as it is already defined in the library), we can remove from the default